test/DartTest.ts

```diff
@@ -165,27 +164,21 @@ describe('transpile to dart', () => {
           .to.equal(' switch ( c ) { case Color . Red : break ; default : break ; }');
     });
     it('does not support const enum', () => {
-      expectErroneousCode('const enum Color { Red }')
-          .to.throw('const enums are not supported');
+      expectErroneousCode('const enum Color { Red }').to.throw('const enums are not supported');
     });
   });
 
   describe('decorators', () => {
-    it('translates plain decorators', () => {
-      expectTranslate('@A class X {}').to.equal(' @ A class X { }');
-    });
-    it('translates arguments', () => {
-      expectTranslate('@A(a, b) class X {}').to.equal(' @ A ( a , b ) class X { }');
-    });
-    it('translates on functions', () => {
-      expectTranslate('@A function f() {}').to.equal(' @ A f ( ) { }');
-    });
-    it('translates on properties', () => {
-      expectTranslate('class X { @A p; }').to.equal(' class X { @ A var p ; }');
-    });
-    it('translates on parameters', () => {
-      expectTranslate('function f (@A p) {}').to.equal(' f ( @ A p ) { }');
-    });
+    it('translates plain decorators',
```

Comment:
hey, no fair! if I run clang-format and get spurious deltas, that's the fault of whoever commited previously.
I'll send a format-only change and rebase this on top.